### ISSUE TITLE
New Skipped tests reporting is wrong in non-verbose mode

### ISSUE THREAD

**Issue by Naktibalda** [Codeception/Codeception#2654]
This issue appeared after upgrading to 2.1.5

`php codecept.phar run tests/unit/PaymentClientCest.php`

```
Unit Tests (6) ------------------------------------------------------------------------------------------------------------
Auth success (PaymentClientCest::authSuccess)                                                                         Skipped
Auth failure (PaymentClientCest::authFailure)                                                                         Skipped
Settle success (PaymentClientCest::settleSuccess)                                                                     Skipped
Settle failure (PaymentClientCest::settleFailure)                                                                     Skipped
Cancel success (PaymentClientCest::cancelSuccess)                                                                     Skipped
Cancel failure (PaymentClientCest::cancelFailure)                                                                     Skipped
---------------------------------------------------------------------------------------------------------------------------


Time: 593 ms, Memory: 11.00Mb

There were 6 skipped tests:

---------
1)
---------
2)
---------
3)
---------
4)
---------
5)
---------
6)
```

The output looks better if I provide -v or --debug parameter (I pasted the relevant part here)
`php codecept.phar run tests/unit/PaymentClientCest.php -v`

```
There were 6 skipped tests:

---------
1) auth success in /.../tests/unit/PaymentClientCest.php: Disabled

---------
2) auth failure in /.../tests/unit/PaymentClientCest.php: Disabled

---------
3) settle success in /.../tests/unit/PaymentClientCest.php: Disabled

---------
4) settle failure in /.../tests/unit/PaymentClientCest.php: Disabled

---------
5) cancel success in /.../tests/unit/PaymentClientCest.php: Disabled

---------
6) cancel failure in /.../tests/unit/PaymentClientCest.php: Disabled

```

**Comment by Naktibalda** [comment]
I can't reproduce this issue in different computer.
I get the skipped tests table, but it looks correct (but different than above) in all modes of verbosity.
It is odd that there is no such table in the travis builds.

```
php codecept.phar run -c codeception/ tests/unit/Codeception/Lib/Driver/PostgresTest.php:testLoadDump
Codeception PHP Testing Framework v2.1.5                                                                                                                                                                                  
Powered by PHPUnit 4.8.21 by Sebastian Bergmann and contributors.                                                                                                                                                         
                                                                                                                                                                                                                          
Unit Tests (9) ------------------------------------------------------------------------------                                                                                                                             
PostgresTest::testLoadDump                                                             Skipped                                                                                                                            
---------------------------------------------------------------------------------------------                                                                                                                             
                                                                                                                                                                                                                          
                                                                                                                                                                                                                          
Time: 254 ms, Memory: 10.50Mb                                                                                                                                                                                             
                                                                                                                                                                                                                          
There was 1 skipped test:

---------
1) PostgresTest::testLoadDump
Coudn't establish connection to database
#1  /.../codeception/tests/unit/Codeception/Lib/Driver/PostgresTest.php:42
#2  /.../codecept.phar:7


```

**Comment by Naktibalda** [comment]
I failed to reproduce it, because I used a different test type.

PostgresTest uses Phpunit's `markTestIncomplete` method.
PaymentClientCest.php uses `@skip` annotation.

**Comment by DavertMik** [comment]
Right now it is fixed in Gherkin branch (I rewrote the output formatter for it)

**Comment by Naktibalda** [comment]
@DavertMik Could you fix it in 2.1 branch?

**Comment by Naktibalda** [comment]
Or we could just make a conclusion that this functionality wasn't complete in 2.1 branch and close it.

This issue didn't appear in 2.1.5, it existed since implementation of @skip annotation (2.1.0 for Cept, 2.1.1 for Cest format)
